contracts/token/Minter.sol

```diff
@@ -107,6 +107,22 @@ contract Minter is Manager, IMinter {
         livepeerToken().burn(_amount);
     }
 
+    /*
+     * @dev Transfer ETH to a recipient
+     * @param _to Recipient address
+     * @param _amount Amount of ETH
+     */
+    function transferETH(address _to, uint256 _amount) external onlyBondingManagerOrJobsManager whenSystemNotPaused {
+        _to.transfer(_amount);
+    }
+
+    /*
+     * @dev Receive ETH from JobsManager
+     */
+    function receiveETH() external payable onlyJobsManager whenSystemNotPaused returns (bool) {
```

Comment:
Since the Minter shouldn't accept ETH from any address except for the JobsManager I thought it would make sense to define this specific function with the modifier restricting access to the JobsManager rather than making the fallback function payable and then doing an additional check there for the sender address